dvc/repo/params/show.py

```diff
@@ -91,7 +94,9 @@ def _read_params(
         fs_paths += [param.fs_path for param in params]
 
     for fs_path in fs_paths:
-        from_path = _read_fs_path(repo.fs, fs_path, onerror=onerror)
+        if Path(fs_path).is_relative_to(repo.root_dir):
```

Comment:
We don’t use pathlib.

Reply:
Do they work if you only change the `fs` to `dvcfs` and keep everything else same?

Reply:
Every filesystem `fs` instances have a `path` object. It depends on what these paths are here, absolute, repo-relative paths or wdir-relative paths, etc.

https://github.com/iterative/dvc/blob/02f7fcae73efc0128ee3ca1fe2ed0fcdf50fc3af/dvc/repo/params/show.py#L33

Reply:
> Do they work if you only change the `fs` to `dvcfs` and keep everything else same?

No, because in the tests, sometimes the paths are relative and sometimes they are absolute. When they are absolute, it raises a FileNotFoundError.

How else can I check whether they are absolute or relative to the repo root, which is what I'm trying to do here?

Reply:
You can use `repo.dvcfs.from_os_path()`, see https://github.com/iterative/dvc/pull/9493/files.

But now it will not work if you are not in root of the repo.